### PR TITLE
Correct tie_embedding_weights and add logit checking

### PR DESCRIPTION
**Pull request by nkovela1**

### CODE REVIEW

File: keras_nlp/models/t5/t5_presets.py [hunk 1]
```diff
@@ -36,6 +36,7 @@
             "activation": "relu",
             "use_gated_activation": False,
             "layer_norm_epsilon": 1e-06,
+            "tie_embedding_weights": True,
```
Comment (mattdangerw) [inline keras_nlp/models/t5/t5_presets.py:39]: let's update the default for T5Backbone to `True`

**Review by mattdangerw — APPROVED** [review]
looks good!

File: keras_nlp/models/t5/t5_presets.py (hunk 1 above)
Comment (nkovela1) [reply keras_nlp/models/t5/t5_presets.py:39]: Changed the default, thanks!

**Comment by mattdangerw** [comment]
Looks like still some hash matching issues.

**Comment by nkovela1** [comment]
Fixed the hash! (one digit missing)

**Pull request merged**